app/assets/javascripts/hyrax.js

```diff
@@ -70,10 +70,7 @@
 //= require hyrax/permissions
 //= require hyrax/notifications
 //= require hyrax/autocomplete
-//= require hyrax/autocomplete/location
-//= require hyrax/autocomplete/subject
-//= require hyrax/autocomplete/language
-//= require hyrax/autocomplete/work
+//= require_tree ./hyrax/autocomplete/
```

Comment:
I'd prefer we be explicit here. Is there any reason we need to make this change?